ts/features/pn/analytics/__test__/activationReminderBanner.test.ts

```diff
@@ -0,0 +1,103 @@
+import { mixpanelTrack } from "../../../../mixpanel";
+import { buildEventProperties } from "../../../../utils/analytics";
+import { MESSAGES_ROUTES } from "../../../messages/navigation/routes";
+import PN_ROUTES from "../../navigation/routes";
+import { sendBannerMixpanelEvents } from "../activationReminderBanner";
+
+jest.mock("../../../../mixpanel", () => ({
+  mixpanelTrack: jest.fn()
+}));
+
+jest.mock("../../../../utils/analytics", () => ({
+  buildEventProperties: jest
+    .fn()
+    .mockImplementation((channel, category, properties) => ({
+      channel,
+      category,
+      properties
+    }))
+}));
+
+describe("activationReminderBanner", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should track banner shown event", () => {
+    sendBannerMixpanelEvents.bannerShown();
+
+    expect(mixpanelTrack).toHaveBeenCalledWith(
+      "BANNER",
+      buildEventProperties("UX", "screen_view", {
```

Comment:
It's better to keep tests decoupled from logics that are used inside the original method. If you want to ease your work, instead of using the original function, create an utility one inside this tests that creates the instance that is expected in ouput